bundles/asdi/asdi-projection-change/plugin/ProjectionChangerPlugin.js

```diff
@@ -0,0 +1,101 @@
+Oskari.clazz.define( 'Oskari.projection.change.ProjectionChangerPlugin',
+  function ( config, instance ) {
```

Comment:
I would say a better design would be that you pass localization instead of instance and create the flyout here.